lib/monitoring/uptime.ex

```diff
@@ -15,6 +15,42 @@ defmodule Monitoring.Uptime do
     Logger.info("arinc_status_to_splunk_ms: #{div(splunk_time, 1000)}")
   end
 
+  defp log_node_status(
+         %{
+           "IPAddress" => ip_address,
+           "Uptime" => uptime,
+           "TotalMemory" => total_memory,
+           "Total_C_Space" => total_c_space,
+           "Total_D_Space" => total_d_space,
+           "AvailableMemory" => available_memory,
+           "Available_C_Space" => available_c_space,
+           "Available_D_Space" => available_d_space
+         },
+         date_time
+       ) do
+    Logger.info([
+      "headend_server_stats: ",
+      "date_time=",
+      DateTime.to_string(date_time),
+      " ip_address=",
+      ip_address,
+      " uptime=",
+      uptime,
```

Comment:
If this has spaces, would wrapping it in `inspect` help splunk parse it more easily?